### PR TITLE
Redirect readthedocs to docs.cloud.service.gov.uk

### PR DESCRIPTION
**Pull request by Jonty**
## What

Readthedocs doesn't allow redirections, or custom themes that would allows us to deploy a simpler solution, so this replaces every page in the documentation with a block of text linking to the correct page in the new documentation.

Automatic redirection is possible because RTD allows you to include custom javascript, so this simply grabs the link in the page and changes the window location after 5 seconds.

## How to review

1. Install `mkdocs`
2. Run `mkdocs serve`
3. Visit the URL given, appending URL's from the [old list of URL's](https://gist.github.com/Jonty/09fa57685e4eef1c54398660f0a563df) to the end. You'll need to strip the `/en/latest` bit, as that is added by RTD and won't work with `mkdocs`
4. Check that every URL in the [old list of URL's](https://gist.github.com/Jonty/09fa57685e4eef1c54398660f0a563df) is redirected to the appropriate page.

## Who can review

Anyone, but @benh-gds should probably have a say if this approach is ok.

### CODE REVIEW

**Comment by benh-gds** [comment]
This is excellent. I will test this.

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +0,0 @@
-language: python
-python:
-  - "2.7"
-install: "pip install -r requirements.txt"
-script: ./check_links.sh
```
Comment (alext) [inline .travis.yml]: Would it not make sense to leave this in place to verify that all the redirect links are valid?

Comment (Jonty) [reply .travis.yml]: 1. All the redirect links are to fragment URL's, so they won't really get validated.
2. There's no index or link tree to crawl anymore, just pages for each URL that do not refer to one another.

**Comment by benh-gds** [comment]
I'm having trouble getting the readthedocs platform to actually apply the custom theme (in a test account). I've got this far: http://paas-developer-docs-test.readthedocs.io/en/redirect_all_the_things/ 

but it's not applying the theme. Do you know how to do that, @Jonty ?

**Comment by Jonty** [comment]
@benh-gds Digging into this now.

**Comment by Jonty** [comment]
Right, readthedocs basically doesn't allow custom themes for $reasons. I've rewritten this so each file contains the text and a link, and added some custom javascript that causes it to redirect after 5s. 

It's not nearly as clean, but hopefully it should work on RTD.

**Comment by benh-gds** [comment]
Tested, works, merging.

**Comment by dcarley** [comment]
👏

**Pull request merged**